roles/hosted_engine_setup/tasks/bootstrap_local_vm/05_add_host.yml

```diff
@@ -217,6 +217,14 @@
       register: host_result_up_check
       ignore_errors: true
 
+    - name: Enable postfix service
+      ansible.builtin.service:
+        name: postfix
+        enabled: yes
```

Comment:
@shubhaOracle can you please change it to `true` instead of `yes`?

Reply:
Thanks for the review. Made the change.